CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 ### Fixes
 
 - [#2342: Open fewer files at once during migration](https://github.com/alphagov/govuk-prototype-kit/pull/2342)
+- [#2340: Do not compile sass files with an underscore prefix](https://github.com/alphagov/govuk-prototype-kit/pull/2340)
+- [#2336: Fix failure to read manage-prototype.css in the error server](https://github.com/alphagov/govuk-prototype-kit/pull/2336)
```

Comment:
2336 is already in the change log from the previous release.  Please remove.

Reply:
Thanks, removed.